.github/workflows/fetch_zotero.yml

```diff
@@ -24,6 +24,9 @@ on:
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
+  schedule:
+    - cron:  '* 2 * * *'
```

Comment:
From my understanding of this, the current format is updating every minute from 2 am to 2:59 am.  Is this the intended behavior or is it to fetch Zotero updates once every night at 2 am?

Reply:
Good catch!!  I've updated it